src/circleci-cli/orb.yml

```diff
@@ -10,6 +14,7 @@ commands:
             circleci version
             echo "Run circleci help"
             circleci help
+            echo -e "token: $<< parameters.token-env-var >>\nverbose: false" > ~/.circleci/cli.yml
```

Comment:
1. Hvae you considered default to a commonly used name (i.e. CIRCLE_CLI_TOKEN) instead of empty
2. If expecting an env var which can be empty, we should include [a `when` conditional](https://github.com/CircleCI-Public/config-preview-sdk/blob/master/docs/conditional-steps.md#example) so that it only adds to cli.yml if the user provided a value.
3. In either case, some bash logic to ensure the resolved ENVVAR is not empty would be good.